site/docs/v1/tech/apis/identity-providers/index.adoc

```diff
@@ -23,6 +23,10 @@ The type of the identity provider will determine the object's properties as well
 
 To learn how to configure these Identity Providers using the FusionAuth UI, go here link:/docs/v1/tech/identity-providers/[Identity Providers].
 
+=== Link APIs
+
+Links between an Identity Providers and a FusionAuth User are configured in the `linkingStrategy` for each Identity Provider.  To learn more about how `linkingStrategy` works click into any of the Identity Provider API docs listed above.  For working with Identity Provider Links directly there is a separate Links API which is used to create a link between a FusionAuth user and a user in a 3rd party identity provider.  To learn more about working with the Links API, go here link:/docs/v1/tech/apis/identity-providers/links/[Link APIs].
```

Comment:
Suggestion:

> The way a link is established between an identity provider and FusionAuth is determined by the `linkingStrategy for each identity provider. An API is provided to manual link and unlink a user to a 3rd party identity provider. To learn more about managing links between FusionAuth and a 3rd party identity provider, see the link:/docs/v1/tech/apis/identity-providers/links/[Link API].

Reply:
I've updated the copy and did checked that is renders okay.  So, this should be good to go.